deploy/requirements/click.txt

```diff
@@ -16,4 +16,5 @@ uv==0.1.13
 packaging==24.0
 deepdiff==7.0.1
 pydantic==2.7.3
+polling2==0.5.0
```

Comment:
move it to prod.txt if it needed only for running tests

Reply:
Removed ✅ , looks like it was a dependency which is not used anymore.